DataManagementSystem/Agent/RequestOperations/RegisterReplica.py

```diff
@@ -95,9 +96,8 @@ def __call__(self):
               catMaster = catMaster and FileCatalog()._getCatalogConfigDetails(failedCatalog)\
                   .get('Value', {}).get('Master', False)
           # If one targets explicitly a catalog and it fails or if it fails on the master catalog
-          if (
-                  catalogs or catMaster) and (
-                  'file does not exist' in opFile.Error.lower() or 'no such file' in opFile.Error.lower()):
+          if (catalogs or catMaster) and \
+                  ('file does not exist' in opFile.Error.lower() or 'no such file' in opFile.Error.lower()):
```

Comment:
Is it possible to use `errno.ENOENT` here? https://docs.python.org/3/library/errno.html#errno.ENOENT

Reply:
I would be reluctant as this error comes from a non-standard interface... As it works like this, it seems from time to time it is also `File does not exist` This is fragile code!